applications/charter/P4_Apps_WG_charter.mdk

```diff
@@ -42,13 +52,14 @@ we propose the WG produce the following deliverables initially.
 - Specifications for common telemetry-report format and semantics
 - Test applications for the above
```

Comment:
What do we mean by test applications?

Reply:
We mean test programs to verify their implementation of application semantics. Will change the text.